### DISCUSSION TITLE
Amazing!

### DISCUSSION THREAD

**Discussion by riziles** [janosh/matterviz#26] [Show and tell]
This is so cool. You should share on the site showcase in the SvelteKit discord server:

https://kit.svelte.dev/docs/additional-resources#support

**Comment by janosh** [comment]
@riziles Thanks! Maybe you want to share it there? Might be more convincing that it's worth checking out if I don't toot my own horn. 📯 

If you do post it there, feel free to tag me in case people have questions.
